### DISCUSSION TITLE
Can deprecation warnings be silenced?

### DISCUSSION THREAD

**Discussion by javierjulio** [lostisland/faraday#1329] [Help]
First off, thank you for the deprecation warnings! We've just upgraded from 0.17.x to 1.8.0 and seems to be going well. We use one gem that also depends on Faraday but hasn't been updated yet to address the `Faraday::Connection#authorization` deprecation warning so our test suite is rather noisy. Is it possible to disable Faraday deprecation warnings with the 1.8.0 release? I see mentions of an environment variable and module but no longer seem available as of 1.x.

**Comment by iMacTia** [comment]
Hi @javierjulio and thanks for asking. Unfortunately at the moment there's no way to do such a thing.
The mentions you found are probably about the `Deprecate` module, which was introduced for the transition from 0.x to 1.0 and is not currently available in 1.x.
We could of course reintroduce something similar, but I was curious to know if you couldn't simply raise the logger level or disable it completely during your test run? At least this is what I normally do, but maybe this is not an option in your case?
